docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md

```diff
@@ -977,6 +977,33 @@ Instructs `saucectl` to run all classes for the suite _except_ those specified h
 
 ---
 
+### `shard`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+When shard is configured as concurrency, saucectl automatically splits the tests by concurrency so that they can run in parallel.
+
+Selectable values: `concurrency` to shard by concurrency. Remove this field or leave it empty `""` for no sharding.
+
+:::note
+When shard is set for a suite, the [class](#class) setting is automatically ignored.
+:::
+
+```yaml
+suites:
+  - name: "I am sharded"
+    shard: concurrency
+```
+---
+
+### `testListFile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The file containing tests is used in sharding by concurrency.
+
```

Comment:
Should the file format be described here? If not, where can the user find it? If the user searches for this field within the docs, they'll ask themselves that question and start looking around the page.

Reply:
Ok. Will add it.